tsdoc/src/parser/TSDocMessageId.ts

```diff
@@ -339,44 +339,49 @@ export enum TSDocMessageId {
   /**
    * Expecting an attribute or `>` or `/>`.
    */
-  HtmlTagMissingGreaterThan = 'tsdoc-html-tag-missing-greater-than',
+  XmlTagMissingGreaterThan = 'tsdoc-xml-tag-missing-greater-than',
 
   /**
-   * Expecting `=` after HTML attribute name.
+   * Expecting `=` after XML attribute name.
    */
-  HtmlTagMissingEquals = 'tsdoc-html-tag-missing-equals',
+  XmlTagMissingEquals = 'tsdoc-xml-tag-missing-equals',
 
   /**
-   * Expecting an HTML string starting with a single-quote or double-quote character.
+   * Expecting an XML string starting with a single-quote or double-quote character.
    */
-  HtmlTagMissingString = 'tsdoc-html-tag-missing-string',
+  XmlTagMissingString = 'tsdoc-xml-tag-missing-string',
 
   /**
-   * The HTML string is missing its closing quote.
+   * The XML string is missing its closing quote.
    */
-  HtmlStringMissingQuote = 'tsdoc-html-string-missing-quote',
+  XmlStringMissingQuote = 'tsdoc-xml-string-missing-quote',
 
   /**
    * The next character after a closing quote must be spacing or punctuation.
    */
-  TextAfterHtmlString = 'tsdoc-text-after-html-string',
+  TextAfterXmlString = 'tsdoc-text-after-xml-string',
 
   /**
-   * Expecting an HTML tag starting with `</`.
+   * Expecting an XML tag starting with `</`.
    */
-  MissingHtmlEndTag = 'tsdoc-missing-html-end-tag',
+  MissingXmlEndTag = 'tsdoc-missing-xml-end-tag',
+
+  /**
+   * XML element tag is unterminated.
```

Comment:
(Just a suggestion, open for discussion.) The word "unterminated" might be unclear to some readers, particularly if they weren't intending to write XML.

```suggestion
   * The corresponding XML end tag is missing.
```

This error gets a prefix, so the full message would be `Invalid XML element: The corresponding XML end tag is missing.`

If it's not too difficult, we could even include the name of the tag:  `The corresponding end tag "</abcd>" is missing.`  (substitute `abcd` with the actual name).

Reply:
I went ahead and added this with the corresponding tag name in the error: `The corresponding end tag "</abcd>" is missing.`